src/Core/Addon/Module/ModuleManager.php

```diff
@@ -344,15 +332,17 @@ public function upgrade($name, $version = 'latest', $source = null)
      */
     public function disable($name)
     {
-        if (!$this->employee->can('edit', 'AdminModulessf')
-            || !$this->moduleProvider->can('configure', $name)) {
+        if (!Tools::isPHPCLI() && (!$this->employee->can('edit', 'AdminModulessf')
+            || !$this->moduleProvider->can('configure', $name))) {
```

Comment:
This test is duplicated many times (4 times for this snippet)
Should be factored in a function with an explicit name like notPhpCliAndCantEditOrConfigureModule